.github/workflows/dispatch.yaml

```diff
@@ -0,0 +1,16 @@
+name: Simple workflow with dispatch trigger
```

Comment:
Should we add some comments to the top of the file just explaining that is serves the purpose of testing something else?
```suggestion
# ...
# ...
name: Simple workflow with dispatch trigger
```